crates/zkforge/bin/cmd/script/build.rs

```diff
@@ -178,15 +178,11 @@ impl ScriptArgs {
                             .expect("The target specifier is malformed.");
                     let path = std::path::Path::new(&path);
 
-                    // Remove dir prefix for files in script/ dir
-                    let mut new_path = PathBuf::from("");
-                    if path.starts_with("script/") {
-                        for component in path.strip_prefix("script/").unwrap().iter() {
-                            new_path.push(component);
-                        }
-                    } else {
-                        new_path = path.to_path_buf();
-                    }
+                    // Remove dir prefix for files in script/ or scripts/ dir
+                    let new_path = path
+                        .strip_prefix("script/")
+                        .or_else(|_| path.strip_prefix("scripts/"))
+                        .unwrap_or(path);
```

Comment:
The only issue with this is that it really should account for remappings / configuration ? For instance, in `foundry.toml` you can specify:
```
[profile.zksync]
src = 'src'
test = 'tests'
script = 'scripts'
```

`script = 'scripts'` can be also be `script = 'scrip'` for instance?

Reply:
Thanks for the suggestion, it should be fixed with 27f4cb2